libexec/alluxio-config.sh

```diff
@@ -60,6 +55,8 @@ if [[ -n "${ALLUXIO_HOME}" ]]; then
   ALLUXIO_JAVA_OPTS+=" -Dalluxio.home=${ALLUXIO_HOME}"
 fi
 
+ALLUXIO_JAVA_OPTS+=" -Dalluxio.conf.dir=${ALLUXIO_CONF_DIR}"
```

Comment:
Unrelated to this PR, but it seems error prone expecting all `+=` setters to prepend a whitespace.